package.json

```diff
@@ -41,6 +41,7 @@
     "@types/node": "^15.12.2",
```

Comment:
Read the other comments but overall I don't think it's worth continuing on this. Doesn't look like a simple way t do what I suggested unless you want to dig further.

We can keep the part that verifies it's a valid address

Let me know what you think.